src/rust/engine/process_execution/src/local.rs

```diff
@@ -878,7 +882,7 @@ pub fn setup_run_sh_script(
   let full_script = format!(
     "#!/bin/bash
 # This command line should execute the same process as pants did internally.
-export {stringified_env_vars}
+{stringified_env_var_export}
```

Comment:
Not in scope but using:
```
cd {stringified_cwd}
env -i {stringified_env_vars} {stringified_command_line}
```

Would be nicer. We'd finally get sandbox parity by blanking out the env as is done in the scheduler.

Reply:
I can make it in scope, if I'm here anyway. That would both fix the trivial issue and be more substantial.